adapters/teads/teads_types.go

```diff
@@ -0,0 +1,40 @@
+package teads
+
+import (
+	"encoding/json"
+	"text/template"
+)
+
+type adapter struct {
+	endpointTemplate *template.Template
+}
+
+type DefaultBidderImpExtension struct {
+	Bidder Bidder `json:"bidder"`
+}
+
+type Bidder struct {
+	PlacementId int `json:"placementId"`
+}
+
+type TeadsImpExtension struct {
+	KV TeadsKV `json:"kv"`
+}
+
+type TeadsKV struct {
+	PlacementId int `json:"placementId"`
+}
+
+type TeadsBidExt struct {
+	Prebid TeadsPrebidExt `json:"prebid"`
+}
+
+type TeadsPrebidExt struct {
+	Meta TeadsPrebidMeta `json:"meta"`
+}
+
+type TeadsPrebidMeta struct {
```

Comment:
Scope of the  above structs is limited to teads package. Therefore recommended to make them private by starting struct name with lowercase alphabet.

```
example:
- type TeadsPrebidExt struct
+ type teadsPrebidExt struct



```